### DISCUSSION TITLE
How to set user_created to half width but can occupy the entire line?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8835] [Archive]
![image](https://user-images.githubusercontent.com/8030134/137540875-c464d605-baec-4905-a741-c85ad26f11ae.png)

How to set user_created to half width but can occupy the entire line?

**Comment by rijkvanzanten** [comment]
Maybe I'm misunderstanding the question, but you can control the field size in the triple-dot menu:

<img width="501" alt="CleanShot 2021-10-18 at 12 48 13@2x" src="https://user-images.githubusercontent.com/9141017/137774124-cbf1c649-cad7-4378-ab35-72ee1705602c.png">

**Reply by benhaynes** [reply]
I believe they're asking about what we used to call `half_left`, where it's half width, but forces nothing to go to the right. As of now, the only way to do this is to have a `full` width field after, so the two half widths don't collapse to the same line. We'll never have a `half_right`, but could consider adding the `half_left` back ... but that needs more discussion first.

**Reply by rijkvanzanten** [reply]
Ohhh I see!
